app/Http/Controllers/Daemon/ActionController.php

```diff
@@ -4,6 +4,7 @@
 
 use Cache;
 use Illuminate\Http\Request;
+use Pterodactyl\Events\Server\Installed;
```

Comment:
I probably don't do this elsewhere, but can we import this as `ServerInstalledEvent` so when it is used in the code its obvious what we are doing?